packages/scanner/src/cli/scan/watchScan.ts

```diff
@@ -111,14 +112,16 @@ export class Watcher {
     );
   }
 
-  private queue = queue<string>(this.scan.bind(this), 2);
+  // do not remove callbackify, apparently on windows
+  // passing plain async function doesn't work (?)
+  private queue = queue<string>(callbackify(this.scan.bind(this)), 2);
```

Comment:
Do you know why this is?

Reply:
After some digging around, I noticed that only the first N items were being processed on Windows, where N is the concurrency (in our case 2). I googled around and found this [SO post](https://stackoverflow.com/questions/57448099/async-queue-crashes-after-processing-3-initial-elements-of-the-queue-with-concur), and then I added the `callback` to the worker and then all of the items were processed.